build-list.cfg

```diff
@@ -3,6 +3,7 @@
 #
 # NOTE: line must have colon for now even if no plugins - will fix sometime
```

Comment:
- Colons were removed from version lines, so this comment line should be removed or updated to avoid confusion.